vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -1066,6 +1070,9 @@ public UpdateQueueData getUpdateQueueData() {
     private boolean verticalScrollingEnabled = true;
 
     private SerializableFunction<T, String> classNameGenerator = item -> null;
+    private SerializableFunction<T, Boolean> dropFilter = item -> true;
+    private SerializableFunction<T, Boolean> dragFilter = item -> true;
```

Comment:
Change the types of the filters to `SerializablePredicate<T>`

Reply:
Done.